installers/cubeit-installer

```diff
@@ -878,6 +905,9 @@ smart update"
 
 fi
 
+if [ -d "$CURRENTDIR/$PUPPETDIR" ]; then
```

Comment:
I want to support relative path instead of hardcoded path, so here make a test and see if it is a relative one.